envs/monkey_zoo/terraform/monkey_zoo.tf

```diff
@@ -20,6 +20,16 @@ resource "google_compute_network" "tunneling2" {
   auto_create_subnetworks = false
 }
 
+resource "google_compute_network" "credential-reuse" {
+  name = "${local.resource_prefix}credential-reuse"
+  auto_create_subnetworks = false
+}
+
+resource "google_compute_network" "credential-reuse2" {
+  name = "${local.resource_prefix}credential-reuse2"
+  auto_create_subnetworks = false
+}
+
```

Comment:
Already defined on line 57?

Reply:
Those are two different things. Networks and subnetworks.